### DISCUSSION TITLE
Add image in the footer

### DISCUSSION THREAD

**Discussion by Vinc26** [GetPublii/Publii#1770] [Help] [answered]
Hi there,

I want to add an image in the footer (of last Simple Theme). So in the Theme > Footer section, there is a Tiny editor. But there isn't any button to add an image. So I've tried to add it directly in html code :
- Adding an image using the File Manager
- Adding the usual html code, with a screen like `src="media/files/footerimage.jpg"`

The problem is that it works only for the homepage. In other pages, the link is broken. Yes I can use the absolute link: but it doesn't work in local preview.
Is there an another way to add image in footer? Without editing the theme ? And if possible using the responsive loading option (small image for small screen...) ?

:)

**Comment by michaelfluegge** [comment]
in short terms: use your override (the one from your other questions :-) ), put the footer.hbs there or use the existing one, add the code for the image. After that, it should work even I the preview.

**Comment by Vinc26** [comment, marked as answer]
Finally, I've used this solution: https://github.com/GetPublii/Publii/discussions/1751#discussioncomment-11030361
